### PR TITLE
add custom asserts

### PR DESCRIPTION
**Pull request by amiordan**
add clearTextField method
add access modifiers where was needed

### CODE REVIEW

File: src/test/java/stepDefinition/ShoppingCartStepDef.java [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
+public class ShoppingCartStepDef extends AbstractStepDef {
 
-    ShoppingCartPage shoppingCartPage = new ShoppingCartPage(webDriver);
+    private ShoppingCartPage shoppingCartPage = new ShoppingCartPage(webDriver);
 
     @Then("^selected '(.*)' from Wishlist is added successfully to ShoppingCart$")
-    public void clickAddToCartFromWishlist (String expectedItem) {
-        Assert.assertEquals("Selected product is not added to ShoppingCart",expectedItem, shoppingCartPage.getActualItem().getText());
+    public void checkProductAddedToCartFromWishList(String expectedItem) {
+        assertThat("Selected product is dded to ShoppingCart", expectedItem, is(shoppingCartPage.getActualItem().getText()));
```
Comment (vdones) [inline src/test/java/stepDefinition/ShoppingCartStepDef.java]: added
Correct please

File: src/test/java/stepDefinition/WishListPageStepDef.java [hunk 2]
```diff
@@ -1,29 +1,34 @@
...
         String actual = wishListPage.getQuantityField().getAttribute("value");
-        Assert.assertEquals("Quantity does not match", ScenarioContext.getValue("quantity"), actual);
+        assertThat("Quantity does not match", ScenarioContext.getValue("quantity"), is(actual));
     }
 
     @Then("^product is removed successfully from Wishlist$")
     public void productIsRemovedSuccessfully() {
         waitVisibility(wishListPage.getEmptyWishlistMessage());
-        Assert.assertEquals("The wishlist is empty!", wishListPage.getEmptyWishlistMessage().getText());
+        assertTrue("The wishlist is empty", "The wishlist is empty!".equals(wishListPage.getEmptyWishlistMessage().getText()));
```
Comment (vdones) [inline src/test/java/stepDefinition/WishListPageStepDef.java]: Change to assert equals please
Or use assertUtils

File: src/test/java/stepDefinition/ShoppingCartStepDef.java (hunk 1 above)
Comment (amiordan) [reply src/test/java/stepDefinition/ShoppingCartStepDef.java]: done, thank you!

File: src/test/java/stepDefinition/WishListPageStepDef.java (hunk 2 above)
Comment (amiordan) [reply src/test/java/stepDefinition/WishListPageStepDef.java]: done, thank you!

**Review by MaximRaicov — APPROVED** [review]

File: src/test/java/stepDefinition/MyAccountStepDef.java [hunk 3]
```diff
@@ -18,7 +20,7 @@
     public void userChangesField(String elementName, String extension, Map<String, String> data) {
         waitForPageLoaded();
         beforeEdit = getElementByName(elementName.concat(extension)).getAttribute("value");
-        data.forEach((key, value) -> getElementByName(key).clear());
+        data.forEach((key, value) -> clearTextField(getElementByName(key)));
         data.forEach((key, value) -> getElementByName(key).sendKeys(value));
```
Comment (CristianManic) [inline src/test/java/stepDefinition/MyAccountStepDef.java]: as you made wrappers for the others, why would you let **sendKeys()** like this?

File: src/test/java/stepDefinition/SearchPageStepDef.java [hunk 4]
```diff
@@ -14,13 +14,8 @@
 
     @Then("^'(.*)' list of items is displayed$")
     public void searchResultsList(String searchItem) {
-        Assert.assertTrue(getElementByName("searchProductGrid").findElement(By.xpath("//h2[@class='product-title']//a[contains(text(),'" + searchItem + "')]")).isDisplayed());
+        assertTrue("Search product grid is displayed",
+                getElementByName("searchProductGrid").findElement(By.xpath("//h2[@class='product-title']//a[contains(text(),'" + searchItem + "')]")).isDisplayed());
         getPage(getPageObjectName(SearchPage.class));
     }
```
Comment (CristianManic) [inline src/test/java/stepDefinition/SearchPageStepDef.java:20]: why would you keep a step definition class with just one method?

File: src/test/java/stepDefinition/MyAccountStepDef.java (hunk 3 above)
Comment (amiordan) [reply src/test/java/stepDefinition/MyAccountStepDef.java]: done, thank you!

File: src/test/java/stepDefinition/WishListPageStepDef.java [hunk 5]
```diff
@@ -1,29 +1,33 @@
...
         String actual = wishListPage.getQuantityField().getAttribute("value");
-        Assert.assertEquals("Quantity does not match", ScenarioContext.getValue("quantity"), actual);
+        assertThat("Quantity does not match", ScenarioContext.getValue("quantity"), is(actual));
     }
 
     @Then("^product is removed successfully from Wishlist$")
     public void productIsRemovedSuccessfully() {
         waitVisibility(wishListPage.getEmptyWishlistMessage());
-        Assert.assertEquals("The wishlist is empty!", wishListPage.getEmptyWishlistMessage().getText());
+        assertThat("The wishlist is empty", "The wishlist is empty!", is(wishListPage.getEmptyWishlistMessage().getText()));
```
Comment (CristianManic) [inline src/test/java/stepDefinition/WishListPageStepDef.java]: you are not asserting that will list is empty, but that message corresponds to the expected one, so change the log message to a related one

File: src/test/java/stepDefinition/SearchPageStepDef.java (hunk 4 above)
Comment (amiordan) [reply src/test/java/stepDefinition/SearchPageStepDef.java:20]: cause it is related only to SearchPage

File: src/test/java/stepDefinition/WishListPageStepDef.java (hunk 5 above)
Comment (amiordan) [reply src/test/java/stepDefinition/WishListPageStepDef.java]: done, thank you !

**Review by anciuvaga — APPROVED** [review]

**Pull request merged**